.github/CODEOWNERS

```diff
@@ -147,7 +147,6 @@ internal/tools/                                                     @open-teleme
 pkg/batchperresourceattr/                                           @open-telemetry/collector-contrib-approvers @atoulme @dmitryax
 pkg/batchpersignal/                                                 @open-telemetry/collector-contrib-approvers @jpkrohling
 pkg/experimentalmetricmetadata/                                     @open-telemetry/collector-contrib-approvers @rmfitzpatrick
-pkg/datadog/                                                        @open-telemetry/collector-contrib-approvers @mx-psi @dineshg13 @liustanley @songy23 @mackjmr @ankitpatel96
```

Comment:
was this expected?

Reply:
That is the result of running `make update-codeowners`.

Reply:
There's no `metadata.yaml` file for `pkg/datadog`, that's why it would be removed. I assume it was missed in the [introduction of this package](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35067). I think it can be fixed separately, but should be pretty simple to fix here if preferable.

Reply:
`pkg/datadog` is missing a metadata.yaml, I'll add one.